src/localisation/currency.c

```diff
@@ -22,17 +22,17 @@
 #include "string_ids.h"
 
 const currency_descriptor CurrencyDescriptors[CURRENCY_END] = {
-	{ 10	, CURRENCY_PREFIX, "\xC2\xA3"		,	CURRENCY_SUFFIX,	"GBP"	, STR_POUNDS		},		// British Pound
-	{ 10	, CURRENCY_PREFIX, "$"				,	CURRENCY_PREFIX,	"$"		, STR_DOLLARS		},		// US Dollar
-	{ 10	, CURRENCY_SUFFIX, "F"				,	CURRENCY_SUFFIX,	"F"		, STR_FRANC			},		// French Franc
-	{ 10	, CURRENCY_PREFIX, "DM"				,	CURRENCY_PREFIX,	"DM"	, STR_DEUTSCHMARK	},		// Deutschmark
-	{ 1000	, CURRENCY_PREFIX, "\xC2\xA5"		,	CURRENCY_SUFFIX,	"YEN"	, STR_YEN			},		// Japanese Yen
-	{ 10	, CURRENCY_SUFFIX, "Pts"			,	CURRENCY_SUFFIX,	"Pts"	, STR_PESETA		},		// Spanish Peseta
-	{ 1000	, CURRENCY_PREFIX, "L"				,	CURRENCY_PREFIX,	"L"		, STR_LIRA			},		// Italian Lira
-	{ 10	, CURRENCY_PREFIX, "\xC6\x92"		,	CURRENCY_PREFIX,	"fl."	, STR_GUILDERS		},		// Dutch Guilder
-	{ 10	, CURRENCY_SUFFIX, "kr."			,	CURRENCY_SUFFIX,	"kr."	, STR_KRONA			},		// Swedish Krona
-	{ 10	, CURRENCY_PREFIX, "\xE2\x82\xAC"	,	CURRENCY_SUFFIX,	"EUR"	, STR_EUROS			},		// Euro
-	{ 10000 , CURRENCY_PREFIX, "\xE2\x82\xA9"	,	CURRENCY_PREFIX,	"W"		, STR_WON			},		// South Korean Won
-	{ 1000  , CURRENCY_PREFIX, "R "				,	CURRENCY_PREFIX,	"R "	, STR_ROUBLE		},		// Russian Rouble
-	{ 100	, CURRENCY_SUFFIX, " K\xC4\x8D"		,	CURRENCY_SUFFIX,	" Kc"	, STR_CZECH_KORUNA	},		// Czech koruna
+	{ "GBP", 10	, CURRENCY_PREFIX, "\xC2\xA3"		,	CURRENCY_SUFFIX,	"GBP"	, STR_POUNDS		},		// British Pound
+	{ "USD", 10	, CURRENCY_PREFIX, "$"				,	CURRENCY_PREFIX,	"$"		, STR_DOLLARS		},		// US Dollar
+	{ "FRF", 10	, CURRENCY_SUFFIX, "F"				,	CURRENCY_SUFFIX,	"F"		, STR_FRANC			},		// French Franc
+	{ "DEM", 10	, CURRENCY_PREFIX, "DM"				,	CURRENCY_PREFIX,	"DM"	, STR_DEUTSCHMARK	},		// Deutschmark
+	{ "JPY", 1000	, CURRENCY_PREFIX, "\xC2\xA5"		,	CURRENCY_SUFFIX,	"YEN"	, STR_YEN			},		// Japanese Yen
+	{ "ESP", 10	, CURRENCY_SUFFIX, "Pts"			,	CURRENCY_SUFFIX,	"Pts"	, STR_PESETA		},		// Spanish Peseta
+	{ "ITL", 1000	, CURRENCY_PREFIX, "L"				,	CURRENCY_PREFIX,	"L"		, STR_LIRA			},		// Italian Lira
+	{ "NLG", 10	, CURRENCY_PREFIX, "\xC6\x92"		,	CURRENCY_PREFIX,	"fl."	, STR_GUILDERS		},		// Dutch Guilder
+	{ "SEK", 10	, CURRENCY_SUFFIX, "kr."			,	CURRENCY_SUFFIX,	"kr."	, STR_KRONA			},		// Swedish Krona
+	{ "EUR", 10	, CURRENCY_PREFIX, "\xE2\x82\xAC"	,	CURRENCY_SUFFIX,	"EUR"	, STR_EUROS			},		// Euro
+	{ "KRW", 10000 , CURRENCY_PREFIX, "\xE2\x82\xA9"	,	CURRENCY_PREFIX,	"W"		, STR_WON			},		// South Korean Won
+	{ "RUB", 1000  , CURRENCY_PREFIX, "R "				,	CURRENCY_PREFIX,	"R "	, STR_ROUBLE		},		// Russian Rouble
+	{ "CZK", 100	, CURRENCY_SUFFIX, " K\xC4\x8D"		,	CURRENCY_SUFFIX,	" Kc"	, STR_CZECH_KORUNA	},		// Czech koruna
```

Comment:
https://github.com/marijnvdwerf/OpenRCT2/blob/share-currency-detection/src/localisation/currency.c?ts=4

All columns need to line up on tab spacing of 4. Commas should directly proceed the value without space before the comma. Right braces should all line up. Right comments should all line up. The link above shows that it does not do this with tab spacing of 4.